linux/helper_suse.sh

```diff
@@ -28,7 +28,11 @@ add_repo() {
     wget -q "http://${domain}/infrastructure_agent/gpg/newrelic-infra.gpg" -O newrelic-infra.gpg
     rpm --import newrelic-infra.gpg && rm newrelic-infra.gpg
 
-    zypper -n addrepo "http://${domain}/infrastructure_agent/linux/zypp/sles/${VERSION_ID}/x86_64/newrelic-infra.repo"
+    # Pointing to cacheless endpoint if we match the prod URL. Non prod URL will not match
```

Comment:
The comment confuses me a bit. Both paths ( staging and prod ) repos will be fetched and the urls inside will be changed if I understood correctly. 
For the staging one, the base has already the non cached one though
```
baseurl=http://nr-downloads-ohai-staging.s3-website-us-east-1.amazonaws.com/infrastructure_agent/linux/zypp/sles/15.3/x86_64/
gpgkey=https://download.newrelic.com/infrastructure_agent/gpg/newrelic-infra.gpg
```

Reply:
yes, it would not match, but you are right, let's create the repo locally